### PR TITLE
Mobile version works section 

### PR DESCRIPTION
**Pull request by mmsesay**
In the works section I worked on the following:

- Created the layout for the work section
- Added grid layout for the cards
- Create a container for the cards
- Created the cards content, button and implement the hovering effects
- Setup fonts-sizes and font-family

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,23 +18,23 @@
...
-      <p>
+      <p class="text-lg">
         I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hestiate to contact me.
       </p>
-      <div class="sider-icons-menu">
+      <div class="sider-icons-menu text-white font-size-medium">
         <ul>
           <li><i class="fab fa-github icons"></i></li>
           <li><i class="fab fa-linkedin-in"></i></li>
```
Comment (carlos-ssh) [inline index.html]: - The list of icons uses a ul tag to group them, and li per icon; each icon has a link (a tag).

File: index.html [hunk 2]
```diff
@@ -18,23 +18,23 @@
...
+      <h1 class="font-crete-round text-xl primary-text">Hey There. <br>I&apos;m Alex</h1>
       <h5>I am a Software Developer</h5>
-      <p>
+      <p class="text-lg">
         I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
         don’t hestiate to contact me.
       </p>
-      <div class="sider-icons-menu">
+      <div class="sider-icons-menu text-white font-size-medium">
         <ul>
```
Comment (carlos-ssh) [inline index.html:38]: - Close your `` <ul> `` tag.

File: index.html [hunk 3]
```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
...
+          </div>
+          <div class="box">
+            <p class="font-inter">html</p>
+          </div>
+          <div class="box">
+            <p class="font-inter">boostrap</p>
+          </div>
+          <div class="box">
+            <p class="font-inter">Ruby</p>
+          </div>
```
Comment (carlos-ssh) [inline index.html]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

Comment (carlos-ssh) [reply index.html]: -Kindly, these changes are recommended to applied to all items with the same characteristics.

File: index.html [hunk 4]
```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
...
+            <div class="box-container d-flex">
+              <div class="box">
+                <p class="font-inter text-white">html</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">boostrap</p>
+              </div>
+              <div class="box">
+                <p class="font-inter">Ruby</p>
+              </div>
```
Comment (carlos-ssh) [inline index.html]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 5]
```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
...
```
Comment (carlos-ssh) [inline index.html]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 6]
```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
...
```
Comment (carlos-ssh) [inline index.html]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 7]
```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
...
```
Comment (carlos-ssh) [inline index.html]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

File: index.html [hunk 8]
```diff
@@ -48,5 +48,206 @@ <h5>I am a Software Developer</h5>
...
```
Comment (carlos-ssh) [inline index.html]: - The technology categories ("Languages", "Frameworks" and "Skills") are in a ul, each of them in a li.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @mmsesay , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 2 above)
Comment (Emmyn5600) [reply index.html:38]: Well solved

File: index.html (hunk 3 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 4 above)
Comment (Emmyn5600) [reply index.html]: Well solved

File: index.html (hunk 5 above)
Comment (Emmyn5600) [reply index.html]: Well done

File: index.html (hunk 6 above)
Comment (Emmyn5600) [reply index.html]: Well done

File: index.html (hunk 7 above)
Comment (Emmyn5600) [reply index.html]: Well done

File: index.html (hunk 8 above)
Comment (Emmyn5600) [reply index.html]: Well done

**Review by Emmyn5600 — APPROVED** [review]
## STATUS: APPROVED :muscle::clap:

Hi, @mmsesay 

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous code reviewer. Kudos

- To highlight:
   * #### You have the descriptive Readme:heavy_check_mark:
   * #### Your commits look more professional:heavy_check_mark:
   * #### Linters are passing successfully.:heavy_check_mark:
   * #### You have descriptive PR:heavy_check_mark:
   * #### Your project looks awesome:heavy_check_mark:
   
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**